rs/tests/networking/BUILD.bazel

```
@@ -1,5 +1,7 @@
 load("//rs/tests:common.bzl", "BOUNDARY_NODE_GUESTOS_RUNTIME_DEPS", "CANISTER_HTTP_RUNTIME_DEPS", "COUNTER_CANISTER_RUNTIME_DEPS", "DEPENDENCIES", "GRAFANA_RUNTIME_DEPS", "GUESTOS_RUNTIME_DEPS", "MACRO_DEPENDENCIES", "UNIVERSAL_VM_RUNTIME_DEPS")
-load("//rs/tests:system_tests.bzl", "system_test_nns")
+load("//rs/tests:system_tests.bzl", "system_test", "system_test_nns")
+
+CLONER_CANISTER_RUNTIME_DEPS = ["//rs/tests/networkgin:cloner_canister.wasm.gz"]
```

spelling:
```suggestion
CLONER_CANISTER_RUNTIME_DEPS = ["//rs/tests/networking:cloner_canister.wasm.gz"]
```
and export the file from `/rs/tests/networking/BUILD.bazel` as well.